modules/ROOT/pages/guided-setup.adoc

```diff
@@ -66,20 +66,24 @@ Otherwise, provide these app details:
 
 == Configure Users
 
-Give your user the *Exchange Contributor* role:
+Give your user the Exchange contributor permission:
```

Comment:
Nice! I like seeing names replaced by functions in narrative.

Reply:
Great!